examples/pydantic-ai/provider.py

```
@@ -0,0 +1,44 @@
+for evaluation by promptfoo.
+"""
+
+import asyncio
+import os
+from typing import Any, Dict
+
+from agent import run_weather_agent
+
+
+def call_api(
+    prompt: str, options: Dict[str, Any], context: Dict[str, Any]
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)